### DISCUSSION TITLE
Show online/accessibility status of connections

### DISCUSSION THREAD

**Discussion by Gameniti** [1Remote/1Remote#449] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have some machines that I only boot up when I need them. Some other machines are only accessible with VPN activated.

**Describe the solution you'd like**
A way to see the online/accessible status of machines. Something like a small lamp (red for offline, green for online) in the bottom left of the card in card view or next to the name of the connection in list view.
Idk if a simple ping would solve this but if it does add a setting to increase/decrease time between pings.

**Additional context**
I would try to add this but I have almost 0% experience in programming. (especially C#)

**Comment by VShawn** [comment]
Since we have availability detection https://github.com/1Remote/1Remote/issues/420, it seems that this requirement is easy to implement by adding a background timer to check the availability of the addresses. However, we need to consider how to handle the UI and availability of a connection with multiple addresses.

cc @majkinetor

**Comment by VShawn** [comment]
maybe it's better to move the ticket into discussion

**Comment by majkinetor** [comment]
> Since we have availability detection https://github.com/1Remote/1Remote/issues/420, it seems that this requirement is easy to implement by adding a background timer to check the availability of the addresses. However, we need to consider how to handle the UI and availability of a connection with multiple addresses.

Exactly my thoughts. We could have a connection option such as _Test servers availability constantly_ and provide global option for timer which could by default be 1m. Or, we could specify test timer on connection itself so some connections can be tested slower and some faster, but no more then once in minute so we can have single background task running for all connections every minute.

UI should be handled the way we handle disconnections now, with disconnetion icon.  However, looks like system tags should be better option for this (https://github.com/1Remote/1Remote/discussions/416) as that will allow filtering out available connections. 

Multiple IP should be handled in paralell, the same as multiple connections (maybe something more inteligent tho, like to first try on recently available connection and then on others if that one is not available, all in order to minimize the number of running connections).
